packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+#!/bin/bash
```

Comment:
Understood. Please note, that OCF scripts in pacemaker seems always importing an ocf-shellfuncs "library", which has a bash shebang. Although, I have the version of this OCF script 100% /bin/sh compatible, I doubt it makes sense to stick with a "sh" shebang here because of the aforementioned ocf-shellfuncs... But I will update this patch set as well to be shell compatible :)